### DISCUSSION TITLE
slider cross filter mode

### DISCUSSION THREAD

**Discussion by rh-afk** [widgetti/solara#299] [Q&A]
Dear all

is there any mode for slider cross filter for between the value since in [https://solara.dev/api/cross_filter_slider](url)
the mode only several " mode: The mode to use for filtering. Can be one of ==, >=, <=, >, <."

thanks

**Comment by maartenbreddels** [comment]
Hi,

We haven't implemented that. You could look at the current implementation at https://github.com/widgetti/solara/blob/4191fd0dfe3777ca3aebe8b91e3e58c907e49554/solara/components/cross_filter.py#L207

Using the range slider I think the code will look very similar. I'm not sure if it should be put in the same component, or that it should be its own (I think the latter).

**Reply by rh-afk** [reply]
hi @maartenbreddels thanks for your feedback. will look at it and also waiting if you will release a new component so it will make filter in solara more powerful

thanks
